### PR TITLE
Upgrade to php 7.1

### PR DESCRIPTION
**Pull request by Landerstraeten**

### CODE REVIEW

File: src/Collection/ProcessArgumentsCollection.php [hunk 1]
```diff
@@ -14,10 +14,7 @@ public static function forExecutable(string $executable): self
         return new self([$executable]);
     }
 
-    /**
-     * @param string|null $value
-     */
-    public function addOptionalArgument(string $argument, $value)
+    public function addOptionalArgument(string $argument, $value = null): void
```
Comment (SerkanYildiz) [inline src/Collection/ProcessArgumentsCollection.php]:
```suggestion
    public function addOptionalArgument(string $argument, string $value = null): void
```

Comment (Landerstraeten) [reply src/Collection/ProcessArgumentsCollection.php]: @SerkanYildiz $value can also be null or an integer

Comment (SerkanYildiz) [reply src/Collection/ProcessArgumentsCollection.php]: Ok, according to the removed php docblock it should be an string|null, that's why it was my suggested change :)

Comment (Landerstraeten) [reply src/Collection/ProcessArgumentsCollection.php]: What about `?string` and force to cast?

**Pull request merged**